test/unit/modules/blocks/verify.js

```diff
@@ -1942,7 +1986,6 @@ describe('blocks/verify', () => {
 						saveBlock,
 						err => {
 							expect(err).to.be.null;
-							expect(modules.system.update.calledOnce).to.be.true;
```

Comment:
We should still test those.

Reply:
We already cover that case here https://github.com/LiskHQ/lisk/blob/0206f72e80fad817b48e7fe8dc420d54b7af72cb/test/unit/modules/blocks/verify.js#L1941